gaia_dr3_20230707/README.txt

```diff
@@ -0,0 +1,44 @@
+This is a pared-down version of the gaia_dr3_20230707 reference catalog,
+for astrometry with the LSST ci_hsc package, containing the following shards:
+
+    189584 189648
+
+
+Reference Catalog: Gaia DR3
+###########################
+
+Sky coverage: full sky
+Number of sources: 1,811,709,771
+Magnitude range: ~3 - 21 (G magnitude)
+Disk space: 310 GB
+
+Original data: https://www.cosmos.esa.int/web/gaia/dr3
+Contact: Clare Saunders, cmsaunders@princeton.edu Slack: cmsaunders
+
+The full Gaia DR3 catalog in indexed HTM format. This is the first LSST refcat
+to contain the full coordinate covariance.
+
+Citations/acknowledgements
+==========================
+
+Users of this reference catalog should follow the citation and acknowledgement
+instructions from this website:
+https://gea.esac.esa.int/archive/documentation/GDR3/Miscellaneous/sec_credit_and_citation_instructions/
+
+Catalog creation
+================
+
+This refcat was created with a specialized subclass of
+`IngestIndexedReferenceTask` created to handle the gaia instrumental fluxes:
+`IngestGaiaReferenceTask`.
+
+To re-create this catalog, download the complete gaia_source DR2 files from
+the Gaia Archive (~550 GB total) in CSV format:
+http://cdn.gea.esac.esa.int/Gaia/gdr3/gaia_source/
+
+Follow these instructions, noting the substitution of `IngestGaiaReferenceTask`
+for the base-class `IngestIndexedReferenceTask`:
+https://pipelines.lsst.io/v/w_2019_42/modules/lsst.meas.algorithms/creating-a-reference-catalog.html
+
+The configuration that was used to ingest the data is included in this
+directory as `IngestIndexedReferenceTask.py`.
```

Comment:
This sentence is incorrect; the DatasetConfig file is included as `config.py`, but I don't see the convert config included here. It's the one referred to in this section:

https://pipelines.lsst.io/modules/lsst.meas.algorithms/creating-a-reference-catalog.html#write-a-config-for-the-conversion

We should save that along with the refcat in its main directory on USDF, so it gets copied around when people make copies (or did you already do that and just not include it here?). That should maybe be part of the instructions on the above page, and/or in our dataset policy page (see also [DM-31704](https://jira.lsstcorp.org/browse/DM-31704)):

https://developer.lsst.io/usdf/datasets.html#reference-catalogs

Reply:
Thoughts on this comment?

Reply:
Sorry--I missed this comment earlier. I'll copy over the file `convertReferenceCatalogConfig.py` that was used to create the refcat files used here. 

The config is in the Gaia DR3 directory on USDF. I didn't include it here because I pretty-blindly copied the contents of the Gaia DR2 subdirectory.